processing/pcap_filter.py

```diff
@@ -1,18 +1,27 @@
 import logging
 import re
 import os
+from copy import deepcopy
 from scapy.all import *
+conf.verbose = 0 # ask Scapy to be quiet
 
 log = logging.getLogger(__name__)
 
-from cuckoo.common.abstracts import Processing
+try:
+    from cuckoo.common.abstracts import Processing
+except ImportError:
+    # init standalone processing
+    class Processing(object):
+        def __init__(self, pcap_path):
+            self.pcap_path = pcap_path
 
-HOST_IGNORE_LIST = [
+
+HOST_IGNORE_LIST = set([
     '52.179.17.38',  # microsoft ntp server
     '192.88.99.1'
-]
+])
 
-DNS_IGNORE_LIST = [
+DNS_IGNORE_LIST = set([
```

Comment:
logically these belong to the class itself, if the class is initialized every analysis we should move it 
(I think it does because of the `pcap_path`)
this way you don't have to worry about mutating it and can remove lines 76 & 77

Reply:
I think its more readable like that, where the blacklist appears at the head of the document